mobile_files/package.json.orig

```diff
@@ -33,7 +33,7 @@
     "querystring-es3": "0.2.1",
     "react": "16.8.3",
     "react-dom": "16.4.2",
-    "react-native": "git+https://github.com/status-im/react-native.git#status-v0.59.2",
+    "react-native": "0.59.3",
```

Comment:
https://github.com/facebook/react-native/issues/24001 is not included into 0.59.3, wee still have to use our fork

Reply:
Thanks for spotting that! It's a pretty significant regression.

So the path forward is to upgrade our fork to 0.59.3 with the commit that fixes the long press bug? 🤔 

Should I submit PR to status-im/react-native and update this PR accordingly?

Reply:
I've just updated current PR

Reply:
thank you!